testing/tests/DevExpress.exporter/exceljsParts/exceljs.pivotGrid.tests.js

```diff
@@ -869,70 +925,67 @@ QUnit.module('Scenarios', moduleConfig, () => {
         });
     });
 
-    QUnit.test('Export [string(A,B) x None x number]', function(assert) {
+    QUnit.test('Export [string(A,B) x None x None] & showRowGrandTotals', function(assert) {
         const done = assert.async();
         const ds = {
             fields: [
                 { area: 'row', dataField: 'row1' },
-                { area: 'data', summaryType: 'count', dataType: 'number' }
```

Comment:
The initial config was changed significantly: now, there is no 'data' area options and config details are not obvious with new changes.